conf/postgresql.default.conf

```diff
@@ -119,8 +119,8 @@ shared_buffers = 8GB			# min 128kB
 					# (change requires restart)
 # Caution: it is not advisable to set max_prepared_transactions nonzero unless
 # you actively intend to use prepared transactions.
-work_mem = 64MB				# min 64kB
```

Comment:
A bunch of settings seem to have been reverted to default here, but then not set in `conf/postgresql.conf`. Is this deliberate? If so, what is the reasoning?

Reply:
accidental, but related to a current very vague state of how the configurations are actually used